cfgov/unprocessed/css/organisms/form.less

```diff
@@ -0,0 +1,59 @@
+/* topdoc
+  name: Forms
+  family: cf-organisms
+  patterns:
+  - name: Standard form
+    markup: |
+      <form>
+          <h3>Title of form</h3>
+          <p class="o-form-description">Intro text</p>
+          <h3>Description of section</h3>
+          <div class="o-form-section"></div>
+          <div class="o-form-action"></div>
+      </div>
+    codenotes:
+      - |
+        Pattern structure
+        -----------------
+        form
+          h2
+            .o-form-description
+          h3
+            .o-form-section
+          .o-form-actions
+  tags:
+  - cf-organisms
+*/
+
+.o-form-description {
+    margin-top:  unit( @grid_gutter-width / @base-font-size-px, em);
+    margin-bottom:  unit( @grid_gutter-width * 2 / @base-font-size-px, em);
+    .short-desc();
+}
+
+.o-form-section {
+    margin: unit( @grid_gutter-width / @base-font-size-px, em)
+            unit( @grid_gutter-width / -2 / @base-font-size-px, em)
+            unit( @grid_gutter-width * 2/ @base-font-size-px, em)
+            unit( @grid_gutter-width / -2 / @base-font-size-px, em);
+    padding: unit( @grid_gutter-width / @base-font-size-px, em)
+             unit( @grid_gutter-width / 2 / @base-font-size-px, em);
+    background: @gray-5;
+    border: 1px solid @gray-50;
+
+    .respond-to-min(@bp-med-min, {
+        margin-left: unit( -@grid_gutter-width / @base-font-size-px, em);
+        margin-right: unit( -@grid_gutter-width / @base-font-size-px, em);
+    });
+}
+
+.o-form-actions {
+    margin-top: unit( @grid_gutter-width / -2 / @base-font-size-px, em);
+    margin-bottom: unit( @grid_gutter-width * 2/ @base-font-size-px, em);
+
+    .respond-to-max(@bp-sm-max, {
+        .btn {
+            width: 100%;
```

Comment:
It's also used in the hero if the link is a button, but it shouldn't hold up this PR. I've added an issue to capture it at #1307